src/react-app-env.d.ts

```diff
@@ -1 +1,81 @@
 /// <reference types="react-scripts" />
+
+declare module 'react-grid-layout' {
```

Comment:
I haven't found ts typings for the module, so decided to add a naive ad-hoc solution, check out how types and interfaces describe the existing library contract